src/clients/python/crequest.cc

```diff
@@ -29,6 +29,7 @@
 #include <iostream>
 #include "src/clients/c++/request_grpc.h"
 #include "src/clients/c++/request_http.h"
+#include "src/clients/python/shared_memory/shared_memory_handle.h"
```

Comment:
Isn't this the wrapper for the Python only? The request API doesn't need to know anything about it, right?

Reply:
This contains the shared_memory_handle struct used to get the data inside it from the void ptr handle that was returned to python when the shared memory region was created.